packages/fuels-test-helpers/src/lib.rs

```diff
@@ -181,7 +177,7 @@ pub async fn setup_test_client(
     messages: Vec<Message>,
     node_config: Option<Config>,
     chain_config: Option<ChainConfig>,
-) -> (FuelClient, SocketAddr, ConsensusParameters) {
+) -> Result<(FuelClient, SocketAddr, ConsensusParameters), Error> {
```

Comment:
When we try to get a `bound_address` 
```rust
    let bound_address = if requested_port == 0 {
        get_socket_address()
    } else if is_free(requested_port) {
        config.addr
    } else {
        panic!("Error: Address already in use");
    };
```
we could return an error instead of panicking.